dev/bots/docs.sh

```diff
@@ -28,6 +28,11 @@ function generate_docs() {
     # >>> If you update this version, also update it in dev/bots/analyze_sample_code.dart <<<
     "$DART" pub global activate snippets 0.2.3
 
+    # Run the snippets tool once to force building of the package executable,
+    # since "dart pub global run" has issues with startup concurrency.
+    # TODO(gspencergoog): Remove once pub issue is fixed, https://github.com/dart-lang/pub/issues/3165
+    "$DART" pub global run snippets --help > /dev/null
```

Comment:
Maybe not `> /dev/null` so that we can see what is going on?

Reply:
+1

Reply:
OK.  It's only redirecting stdout, not stderr, but I suppose it's worth having some log spam in case there's useful output.